pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -145,6 +146,29 @@ func NewFramework(r Registry, plugins *config.Plugins, args []config.PluginConfi
 		}
 	}
 
+	if plugins.NormalizeScore != nil {
```

Comment:
The idea was to defer the removing of the [NormalizeScore pluginset](https://github.com/kubernetes/kubernetes/blob/e298fc723f2597697b770f88ce47fe93507f7463/pkg/scheduler/apis/config/types.go#L169) from the types, which triggered the API review to a separate PR, leave the logic here the same as we agreed on.

Reply:
Done.